omnigibson/prims/entity_prim.py

```diff
@@ -906,117 +799,28 @@ def get_angular_velocity(self):
-                    self._dc is not None and self._dc.is_simulating():
-                return self.root_link.get_position_orientation()
-            else:
-                return super().get_position_orientation()
-
-    def _set_local_pose_when_simulating(self, translation=None, orientation=None):
-        """
-        Sets prim's pose with respect to the local frame (the prim's parent frame) when simulation is running.
-
-        Args:
-            translation (None or 3-array): if specified, (x,y,z) translation in the local frame of the prim
-                (with respect to its parent prim). Default is None, which means left unchanged.
-            orientation (None or 4-array): if specified, (x,y,z,w) quaternion orientation in the local frame of the prim
-                (with respect to its parent prim). Default is None, which means left unchanged.
-        """
-        current_translation, current_orientation = self.get_local_pose()
-        if translation is None:
-            translation = current_translation
-        if orientation is None:
-            orientation = current_orientation
-        orientation = orientation[[3, 0, 1, 2]]
-        local_transform = tf_matrix_from_pose(translation=translation, orientation=orientation)
-        parent_world_tf = UsdGeom.Xformable(get_prim_parent(self._prim)).ComputeLocalToWorldTransform(
-            Usd.TimeCode.Default()
-        )
-        my_world_transform = np.matmul(parent_world_tf, local_transform)
-        transform = Gf.Transform()
-        transform.SetMatrix(Gf.Matrix4d(np.transpose(my_world_transform)))
-        calculated_position = transform.GetTranslation()
-        calculated_orientation = transform.GetRotation().GetQuat()
-        self.set_position_orientation(
-            position=np.array(calculated_position),
-            orientation=gf_quat_to_np_array(calculated_orientation)[[1, 2, 3, 0]],
-        )
-
-    def set_local_pose(self, translation=None, orientation=None):
-        if self._prim_type == PrimType.CLOTH:
-            if self._dc is not None and self._dc.is_simulating():
-                self._set_local_pose_when_simulating(translation=translation, orientation=orientation)
-            else:
-                super().set_local_pose(translation=translation, orientation=orientation)
-        else:
-            if self._root_handle is not None and self._root_handle != _dynamic_control.INVALID_HANDLE and \
-                    self._dc is not None and self._dc.is_simulating():
-                self._set_local_pose_when_simulating(translation=translation, orientation=orientation)
-            else:
-                super().set_local_pose(translation=translation, orientation=orientation)
+        positions, orientations = self._articulation_view.get_world_poses()
+        return positions[0], orientations[0]
+
+    def set_local_pose(self, position=None, orientation=None):
```

Comment:
this is intentionally overriding the super method, right? should we preserve the syntax (i.e.: use `translation` instead of `position`)?